### ISSUE TITLE
openPopupWidget() scrolls page to top

### ISSUE THREAD

**Issue by drummerjolev** [tcampb/react-calendly#99]
Hey! Great work with this component, it's so useful.

Could you add an option to disable auto-setting the `overflow` style property? https://github.com/tcampb/react-calendly/blob/0d342eb214e5530c504e0536e63ec57a8500b96b/src/calendly-widget.ts#L172-L174

It's causing my main component on the page to autoscroll to the top. I added `overflow: auto !important` to my `<body>` tag for now, however that's a hack I'd like to remove in the future.

**Comment by tcampb** [comment]
Thanks for the feedback @drummerjolev! This is definitely something I would like to change; unfortunately, it is getting quite difficult to continue making changes to the embed script provided by Calendly. As a result, this change will likely not be implemented until I convert the existing Calendly embed script to React.

**Comment by drummerjolev** [comment]
Sounds good, thanks for the update @tcampb! Looking forward to the React component.

**Comment by tcampb** [comment]
@drummerjolev - the changes above are now available for testing in version [`3.0.0-beta.1`](https://github.com/tcampb/react-calendly/tree/v3.0.0-beta.1), which can be installed using the following command:

```bash
npm i react-calendly@3.0.0-beta.1
```

This new version replaces the existing `openPopupWidget` function (which was provided by the Calendly script) with a `PopupModal` component (see example below). As a result, the library no longer changes the `body` tag styles automatically.

```tsx
import { PopupModal } from "react-calendly";

class CustomButtonExample extends React.Component {
  constructor(props) {
    super(props);

    this.state = {
      isOpen: false,
    };
  }

  render() {
    return (
      <div>
        <button
          style={{ display: "block", margin: "0 auto" }}
          onClick={() => this.setState({ isOpen: true })}
        >
          Custom Button
        </button>
        <PopupModal
          url="https://calendly.com/acmesales"
          pageSettings={this.props.pageSettings}
          utm={this.props.utm}
          prefill={this.props.prefill}
          onModalClose={() => this.setState({ isOpen: false })}
          open={this.state.isOpen}
          /*
           * react-calendly uses React's Portal feature (https://reactjs.org/docs/portals.html) to render the popup modal. As a result, you'll need to
           * specify the rootElement property to ensure that the modal is inserted into the correct domNode.
           */
          rootElement={document.getElementById("root")}
        />
      </div>
    );
  }
}
```
